src/analysis/imports.rs

```diff
@@ -119,21 +120,37 @@ impl Imports {
         self.defaults.clear();
     }
 
+    fn common_checks(&self, name: &str) -> bool {
```

Comment:
Document what this function actually does :) When does it return `true` / `false`?